qa/t/test_qa_contact.t

```diff
@@ -195,9 +195,6 @@ $sel->click_ok("commit");
 # The user is no longer the QA contact, and he has no other role
 # with the bug. He can no longer see it.
 
-$sel->wait_for_page_to_load_ok(WAIT_TIME);
```

Comment:
This test tests that if the user removes themselves in a way that they could no longer see the bug that they would see the 'Changes made to this bug ...' on the resulting Access Denied page. With the redirect flow, it's not as easy to do this since I there is a short circut somewhere beforehand that immediately renders the Access Denied page before show_bug.cgi has a chance to render the bug change results. I think removing this test is OK, we still test that the user can't see the page right below this.